packages/ui/constants/application.ts

```diff
@@ -1 +1,3 @@
-export {}
+/* Related to the overall application including API keys, ids, urls */
+
+export const APP_NAME = 'OpenGT';
```

Comment:
Awesome! This is important because it will allow other users of RareCamp to rebrand the application

Reply:
🙏